sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsDelete.java

```diff
@@ -53,7 +53,7 @@ default boolean canDeleteWhere(Filter[] filters) {
 
   /**
    * Delete data from a data source table that matches filter expressions. Note that this method
-   * will be invoked only if {@link #canDeleteWhere(Filter[])} returns true.
+   * will be invoked iff {@link #canDeleteWhere(Filter[])} returns true.
```

Comment:
Spell out "if and only if" , but that is not the same as 'only if', so I wouldn't